src/keys.ts

```diff
@@ -27,6 +28,7 @@ export function isArrowKey(keyCode: string) {
   );
 }
 
-export const getResizeCenterPointKey = (event: MouseEvent) => event.altKey;
+export const getResizeCenterPointKey = (event: MouseEvent | KeyboardEvent) =>
+  event.altKey || event.which === KEYS.ALT_KEY_CODE;
```

Comment:
@jquintozamora why did you introduce `event.which` in this commit? We've recently removed this and I'm not sure if this was fixing some case.